### ISSUE TITLE
Implement stat

### ISSUE THREAD

**Issue by GrayJack** [GrayJack/coreutils#48]

**Comment by Larisho** [comment]
I want to claim this issue as well!

**Comment by GrayJack** [comment]
Yes, sure, just remember to develop on another branch other than master, create a new one for every pull request, just for the sake of good practices :smiley: 

Have fun, doing this one
